index.html

```diff
@@ -0,0 +1,167 @@
+                                adipisicing elit. Officia quia asperiores dicta error</p>
+                        </a></li>
+                </ul>
+                <input type="button" value="Join Made in Lagos Wizkid Fest 2023" id="joinBtn">
+            </section>
+
+            <!--DYNAMIC SPEAKERS WINDOW-->
+            <section class="speakers">
+                <h4>Featured Talents</h4>
+                <ul class="speakerCards" id="speakerSection">
+
+                </ul>
+                <button class="dropdown" id="drop">MORE <span class="drop">&#8744;</span></button>
+                <div class="speakersFull" id="speakersFull">
+                    <ul class="speakerCards" id="speakerFullSection">
+                    </ul>
+                </div>
+            </section>
+        </section>
+        <section id="aboutpage">
+            <section class="welcome-section">
+                <p class="hello">"Hello Nigeria!"</p>
+                <h1>"MADE IN LAGOS" WIZKID FEST 2023</h1>
+                <p class="welcomeP aboutP">The Music Fest brings together the community <br> of experts,professionals
+                    and
+                    talented
+                    musicians home and away <br>we host the Music Fest concert in a <br> different country every two
+                    years and this year we are bringing it home!.
+                    This
+                    year, we hope to expand our invitation list including <br> organizations and individuals.</p>
+                <p class="welcomeP welcomeP2 aboutP1">Please contact us via Email for any further questions about the
+                    Made
+                    in Lagos Music Fest 2023!</p>
+                <a href="#" id="email">lanzzd191@gmail.com</a>
+            </section>
+            <section class="praises" id="praises">
+                <h3>"Made i Lagos" <br> Wizkid Fest 2023 Logo</h3>
+                <p class="welcomeP praisesP">The logo of CC Global Summit was decided through the logo competition from
+                    8. June
+                    to 7. July</p>
+                <a href="#mainpage" class="logo mobilelogo" id="logo"></a>
+                <section class="previousSummitCard">
+                    <h3>See the past "Made in Lagos" Music Fests</h3>
+                    <p>Take a look at the two "Made in Lagos" Music Fest which took place in Nigeria and the United
+                        Kingdom</p>
+                    <ul class="reference">
+                        <li class="fst">
+                            <h2>2021</h2>
+                            <p>"Made in Lagos" Music Fest in Victoria Island</p>
+                        </li>
+                        <li class="lst">
+                            <h2>2018</h2>
+                            <p>"Made in Lagos" Wizkid Fest at the O2 Arena</p>
+                        </li>
+                    </ul>
+                </section>
```

Comment:
- [ ] Kindly style this section to match the given design. Reduce the width of the orange divider and add some spacing.

Your design
![image](https://user-images.githubusercontent.com/81932478/217584176-9c13f877-8c1c-46d0-9ab4-c44b47181f71.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/217584352-26c55a15-20b2-42de-9b1a-e16c2138370f.png)